.github/ISSUE_TEMPLATE/01-compatible-certification-request.md

```diff
@@ -1,5 +1,5 @@
 ---
-[Implementation Name]: Compatible Certification Request
+name: Compatible Certification Request
```

Comment:
+ name: [Implementation Name] Compatible Certification Request

Reply:
actually, this will be only used in the new issue menu. the `title` will be the proposed issue title. So we should probably change title to `title: [Implementation Name]: Compatible Certification Request for MicroProfile OpenTracing [Version]`.